web/js/containers/sidebar/events.js

```diff
@@ -93,6 +103,15 @@ class Events extends React.Component {
       : hasRequestError
         ? 'There has been an ERROR retrieving events from the EONET events API'
         : '';
+
+    let scrollBarVerticalTop = 0;
+    if (events && selected.id) {
```

Comment:
Looks like it doesn't get the right index in polar projections

```suggestion
    if (visibleEvents && selected.id) {
      var index = Object.keys(visibleEvents).indexOf(selected.id);
      // 12 === li total top/bottom padding
      // 32.2 === li height (varies slightly, Chrome 100% browser zoom height used)
      scrollBarVerticalTop = index ? index * (12 + 32.2) : 0;
    }
```

Reply:
Good fix, I'll get that pushed up.